### PR TITLE
Updating the Sphinx Documentation for SimpleITK 2.0

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

File: docs/source/building.rst [hunk 1]
```diff
@@ -4,10 +4,10 @@ Building SimpleITK
 ******************
 
 
-In many cases a user does not need to build SimpleITK because of the
-available pre-built binaries (see :ref:`Downloading the binaries
-<installation-binaries>`). However there are several reasons a user
-might prefer to **build SimpleITK from source code**:
+Pre-built binaries (see :ref:`Downloading the binaries
+<installation-binaries>`) are available in many cases so
+users do not need to built SimpleITK. However, there are
```
Comment (zivy) [inline docs/source/building.rst:9]: built -> build

File: docs/source/building.rst [hunk 2]
```diff
@@ -263,11 +262,11 @@ To install a built python package into the system Python, as root run:
...
  python setup.py install
 
-Alternatively, a Python virtual environment can be created and the
+Preferably, a Python virtual environment is created and the
 distribution installed there.
 
-A Python Wheel file (.whl) can be created in the "Wrapping/Python/dist"
-directory, by building the "dist" target. If you have used the
+A Python Wheel file (.whl) is built by the "dist" target and produced in the
+"Wrapping/Python/dist" directory. If you uses the
```
Comment (zivy) [inline docs/source/building.rst:269]: uses -> use

File: docs/source/building.rst (hunk 1 above)
Comment (blowekamp) [reply docs/source/building.rst:9]: done

File: docs/source/building.rst (hunk 2 above)
Comment (blowekamp) [reply docs/source/building.rst:269]: done

**Pull request merged**